### PR TITLE
PR for public preview changes

### PR DESCRIPTION
**Pull request by vvs11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,13 +41,23 @@ Follow these steps to run your existing Playwright test suite with the service.
...
-1. In the [Playwright portal](https://aka.ms/mpt/portal), select **Generate token** to create the access token.
+### Install Microsoft Playwright Testing package 
 
-    ![Generate access token](https://github.com/microsoft/playwright-testing-service/assets/12104064/2368ad52-d919-4c8a-b916-bdfddbd7a396)
+To use the service, install Microsoft Playwright Testing package. 
+
+```npm
+npm init @azure/microsoft-playwright-testing
+```
+If you already have a `playwright.service.config.ts` file, the package will ask you to override it.
```
Comment (puagarwa) [inline README.md:52]: should we mention, it will generate playwright.service.config.ts for clarity ?

File: README.md [hunk 2]
```diff
@@ -58,7 +69,7 @@ Follow these steps to run your existing Playwright test suite with the service.
 
 ### Set up environment
 
-Ensure that the `PLAYWRIGHT_SERVICE_ACCESS_TOKEN` and `PLAYWRIGHT_SERVICE_URL` that you obtained in previous steps are available in your environment.
+Ensure that the `PLAYWRIGHT_SERVICE_URL` that you obtained in previous step is available in your environment.
```
Comment (puagarwa) [inline README.md:72]: Should we add note for old users, they may have very old URL which is without workspaceId, that wont work ?

File: README.md [hunk 3]
```diff
@@ -71,78 +82,39 @@ npm i --save-dev dotenv
...
-- Override timeouts for service operations, if needed.
+**NOTE**: If you are a part of multiple Microsoft Entra tenants, make sure you sign-in to the tenant where your workspace belongs. You can get the tenant id from Azure portal, see [Find your Microsoft Entra Tenant](https://learn.microsoft.com/azure/azure-portal/get-subscription-tenant-id#find-your-microsoft-entra-tenant). Once you get the ID, sign-in using the command `az login --tenant <TenantID>`
 
-> [!NOTE]
-> Make sure your project uses @playwright/test version 1.37 or above.
+#### Set up authentication using access tokens
+
+You can generate an access token from your Playwright Testing workspace and use it in your setup. However, we strongly recommend Microsoft Entra ID for authentication due to its enhanced security. Access tokens, while convenient, function like long-lived passwords and are more susceptible to being compromised.
+
+1. To use access token based authentication, [Enable access-token based authentication](https://aka.ms/mpt/authentication)
```
Comment (puagarwa) [inline README.md:113]: should we add clarification that, its disabled by default hence need to be enabled.

File: samples/get-started/.github/workflows/get-started.yml [hunk 4]
```diff
@@ -15,14 +18,27 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
-        node-version: 16
+        node-version: 18
+     # This step is to sign-in to Azure to run tests from GitHub Action workflow, https://docs.github.com/en/actions/security-for-github-actions/security-hardening-your-deployments/configuring-openid-connect-in-azure
+    - name: OIDC Login to Azure Public Cloud with AzPowershell (enableAzPSSession true)
```
Comment (puagarwa) [inline samples/get-started/.github/workflows/get-started.yml:23]: https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Clinux#use-the-azure-login-action-with-openid-connect this is more comprehensive guide to setup azure auth from github action, basically it support 2 method and one of them is OIDC but second method via [service connection](https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure-secret) is also available, its possible that azure customer are comfortable with the old one hence lets mention here

Comment (puagarwa) [inline samples/get-started/.github/workflows/get-started.yml:23]: not user of OIDC is clear enough for customers.

**Review by puagarwa — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (vvs11) [reply README.md:76]: Added

**Pull request merged**